bundles/AdminBundle/Resources/public/css/admin.css

```diff
@@ -2036,4 +2036,27 @@ ul.leaflet-draw-actions.leaflet-draw-actions-top.leaflet-draw-actions-bottom li:
     padding: 0px 5px !important;
 }
 
-/** END workflow engine **/
\ No newline at end of file
+/** END workflow engine **/
+
+/** notifications **/
+
+#notification_value {
+    background-color: #fa3e3e;
+    border-radius: 2px;
+    color: #fff;
+    font-size: 10px;
+    line-height: 1.3;
+    min-height: 13px;
+    padding: 1px 3px;
+    background-clip: padding-box;
+    display: inline-block;
+    position: relative;
+    right: -35px;
+    bottom: -25px;
+}
+
+.pimcore_status_notification {
+    background: url(/bundles/pimcoreadmin/img/flat-color-icons/email.svg) center center no-repeat !important;
```

Comment:
we need to find a different icon... this icon is used for mail settings notification and as a result might cause confusion. 
what about sms.svg?

Reply:
I used two icons: comments (for notifications) and sms for (send notifications), what do you think about it? :)